scripts/src/routes/offers.ts

```diff
@@ -1,41 +1,22 @@
 import { Router } from "express";
 
-import { createOrder, getOffers } from "../services/offers";
+import { getOffers } from "../services/offers";
+import { createOrder } from "../services/orders";
 
 export const router: Router = Router();
 
 /**
  * Return a list of offers
  */
 router.get("/", async (req, res, next) => {
-	const options = {
-	};
-
-	try {
-		const result = await getOffers(options);
-		res.status(result.code || 200).send(result.data);
-	} catch (err) {
-		return res.status(500).send({
-			error: "Server Error",
-			status: 500,
-		});
-	}
+	const data = await getOffers({});
```

Comment:
maybe annotate `data` with the type that is returned from `getOffers`.
while the compiler infers it, it's not clear when looking at the code.